packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -422,11 +422,10 @@ export function useGridEditRows(
   );
 
   const handleCellEditStop = React.useCallback(
-    (params: GridCellParams, event?: React.SyntheticEvent) => {
+    (params: GridCellParams, event: React.SyntheticEvent | {}) => {
```

Comment:
@m4theushw do you agree that we now have a default value so the event is never null ?
Otherwise it would crash in `useGridApIEventHandler`